testsuite/MDAnalysisTests/coordinates/test_rdkit.py

```diff
@@ -24,24 +24,44 @@
 
 import pytest
 import MDAnalysis as mda
+from MDAnalysis.topology.guessers import guess_atom_element
 import numpy as np
 from numpy.testing import (assert_equal,
                            assert_almost_equal)
 
-from MDAnalysisTests.datafiles import mol2_molecule
+from MDAnalysisTests.datafiles import mol2_molecule, PDB_full
 
-Chem = pytest.importorskip("rdkit.Chem")
-AllChem = pytest.importorskip("rdkit.Chem.AllChem")
+try:
+    from rdkit import Chem
+    from rdkit.Chem import AllChem
+    from MDAnalysis.coordinates.RDKit import (
+        RDATTRIBUTES, _add_mda_attr_to_rdkit)
+except ImportError:
+    rdkit_installed = False
 
-def mol2_mol():
-    return Chem.MolFromMol2File(mol2_molecule, removeHs=False)
+    def mol2_mol():
+        pass
 
-def smiles_mol():
-    mol = Chem.MolFromSmiles("CCO")
-    mol = Chem.AddHs(mol)
-    cids = AllChem.EmbedMultipleConfs(mol, numConfs=3)
-    return mol
+    def smiles_mol():
+        pass
+else:
+    rdkit_installed = True
 
+    def mol2_mol():
+        return Chem.MolFromMol2File(mol2_molecule, removeHs=False)
+
+    def smiles_mol():
+        mol = Chem.MolFromSmiles("CN1C=NC2=C1C(=O)N(C(=O)N2C)C")
+        mol = Chem.AddHs(mol)
+        cids = AllChem.EmbedMultipleConfs(mol, numConfs=3)
+        return mol
+
+
+requires_rdkit = pytest.mark.skipif(rdkit_installed == False,
```

Comment:
We recently added "import_not_available" to the test utils, it could probably replace this (see test_msd for some of the usage).